docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -120,6 +119,9 @@ The AWS JDBC Driver has several built-in plugins that are available to use. Plea
 | [Read Write Splitting Plugin](./using-plugins/UsingTheReadWriteSplittingPlugin.md)             | `readWriteSplitting`      | Aurora                 | Enables read write splitting functionality where users can switch between database reader and writer instances.                                                                                                                                                                                                                                                                                                                                                                                                        |
 | [Developer Plugin](./using-plugins/UsingTheDeveloperPlugin.md)                                 | `dev`                     | Any database           | Helps developers test various everyday scenarios including rare events like network outages and database cluster failover. The plugin allows injecting and raising an expected exception, then verifying how applications handle it.                                                                                                                                                                                                                                                                                   |
 
+:exclamation: **NOTE**: Aurora Host List Connection Plugin is deprecated. The wrapper is able to automatically set the Aurora host list provider for connections to Aurora MySQL and Aurora PostgreSQL databases.
```

Comment:
Is it automatically or does the user needs to do something to get it? If so, we need to mention it here

Reply:
Users shouldn't need to do anything